plugins/infinispan-snapshot/src/main/resources/server-configuration/jgroups/example-jgroups-tcp.xml

```diff
@@ -0,0 +1,62 @@
+<config xmlns="urn:org:jgroups"
```

Comment:
Not sure if this is possible, but it would be nice to use the default-configs that are provided for JGroups here: https://github.com/infinispan/infinispan/tree/master/core/src/main/resources/default-configs I *think* these are the same for embedded and server (Maybe not the `UFC|MFC_NB` protocols, but we should discuss getting this into ISPN10 with Tristan. (Can't tag him here, unfortunately)

Reply:
@alanfx Which do you prefer (tcp, upd) to be the default?

Reply:
Hey @gustavolira, I'm fine if TCP is the default, I'm more concerned with using our default recommended configurations by default. I added Tristan to the repo, so we can talk about this when he gets back. Did you compare the server and embedded configs to see if they are the same?

Reply:
@alanfx All of our benchmarks are on jdg-qe project, I can keep [TCP](https://github.com/infinispan/infinispan/blob/master/core/src/main/resources/default-configs/default-jgroups-tcp.xml) default on radargun project and on jdg-qe we can still using our default jgroups configuration, I will submit another PR for jdg-qe with new benchmarks to work with serverng, sounds good?